client/src/common/components/MessageBox.jsx

```diff
@@ -15,14 +15,15 @@ const MessageBox = ({ message, type }) => (
       'message-box--gray': type === MessageType.INFO
     })}
   >
-    <span>
+    <span className="message-box__icon">
       <InfoIcon />
     </span>
-    <p className="message-box__content">{message}</p>
+    <p className="message-box__content">{(message, children)}</p>
```

Comment:
Please use `message || children` as the comma operator can be tricky to understand. If the component is used only in the single place below, you can drop the `message` parameter altogether.